cmd/put.go

```diff
@@ -47,6 +47,9 @@ func init() {
 	cmd.Flags().StringVarP(&filename, "file", "f", "", "json file to read key/value pairs from")
 	cmd.Flags().StringVarP(&timeout, "timeout", "t", "1", "timeout in minutes for put")
 	cmd.Flags().BoolVarP(&delete, "delete", "d", false, "deletes parameters that are not present in the json file")
-	cmd.MarkFlagRequired("file")
+	err := cmd.MarkFlagRequired("file")
+	if err != nil {
```

Comment:
my emacs is complaining about this... but maybe this check isn't needed? it's weird... `go vet` does not trigger on this on the `main` branch...

Reply:
This is a good change. If you look into what MarkFlagRequired does, you end up here:

https://github.com/spf13/pflag/blob/v1.0.5/flag.go#L493

If, say, we deleted line 47 so that there was no longer a "file" flag, this call to MarkFlagRequired would return an error, and we wouldn't want to ignore that.